app/forms/form.rb

```diff
@@ -7,6 +7,8 @@ class Form
   attribute :journey
   attribute :params
 
+  delegate :persisted?, to: :claim
```

Comment:
This is `persisted` check is now a method on the base from class so we can lose this delegation